src/build/arg/mod.rs

```diff
@@ -2265,16 +2265,34 @@ impl<'help> Arg<'help> {
     /// [`ArgMatches::is_present`]: ./struct.ArgMatches.html#method.is_present
     /// [`Arg::default_value_if`]: ./struct.Arg.html#method.default_value_if
     pub fn default_value(self, val: &'help str) -> Self {
-        self.default_value_os(OsStr::from_bytes(val.as_bytes()))
+        self.default_values_os(&[OsStr::from_bytes(val.as_bytes())])
     }
 
     /// Provides a default value in the exact same manner as [`Arg::default_value`]
     /// only using [`OsStr`]s instead.
     /// [`Arg::default_value`]: ./struct.Arg.html#method.default_value
     /// [`OsStr`]: https://doc.rust-lang.org/std/ffi/struct.OsStr.html
-    pub fn default_value_os(mut self, val: &'help OsStr) -> Self {
+    pub fn default_value_os(self, val: &'help OsStr) -> Self {
+        self.default_values_os(&[val])
```

Comment:
I don't get this. Is this supposed to be calling itself or did I miss something?

Reply:
It is using the new method `default_values(self, vals: &[&str])` by creating a slice with a single element.

Reply:
The difference between "value" (singular) and "value**s**" (plural)... a nostalgia about wasted hours...